src/OrderChocolatCake.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+import RestaurantButton from "./RestaurantButton";
+
+const OrderChocolateCake = () => {
+  //const types=["Pizza","Salad","Chocolate cake"]
+  const [order, setOrder] = useState(0);
+
+  function orderOne() {
```

Comment:
State and `onClick` function work very well together - well done! But please refactor your code so there is only one reusable `Order` component (not 3 different components) and make use of props🙂 If you pass `orderType` prop to this component, you will be able to replace hard coded ChocolateCake on line 17 with that prop.

And please delete all comments🙂